src/main/assembly/dist/cfg/config.yml

```diff
@@ -27,6 +27,20 @@ server:
       port: 20301
 
 ingestFlow:
+  skipHidden:
+    # see also dd-dtap/provisioning/group_vars/
```

Comment:
Make the list empty by default. The correct list should be configured by `dd-dtap`.